### PR TITLE
new plugin: wireless network toggles

### PR DESCRIPTION
**Pull request by Nxt3**
Easily toggle wifi or bluetooth!

### CODE REVIEW

**Comment by nate-parrott** [comment]
thanks! this looks really well-executed.

**Comment by Nxt3** [comment]
No problem! Thank you for Flashlight.

File: PluginDirectories/1/wireless-toggles.bundle/info.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "description" : "Easily toggle wifi/bluetooth",
+  "categories" : ["System"],
+  "examples" : [
+    "wifi on",
+    "turn wifi on",
+    "turn bluetooth on",
+    "bluetooth off"
+  ],
+  "creator_name": "Nate Gantt",
+  "creator_url": "TEMP PLACEHOLDER BEFORE THIS GOES ON GITHUB"
```
Comment (nate-parrott) [inline PluginDirectories/1/wireless-toggles.bundle/info.json]: want to change this? :)

**Comment by Nxt3** [comment]
Oops. Well, originally I thought I would make it open-source--but then I remembered where I had to add it, lol. I was just going to put the URL to the repo; not needed anymore. You can delete it or not worry about it until I update something. ;p

**Pull request merged**